data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -3365,6 +3453,28 @@ spec:
                     maxItems: 2
                     type: array
                     uniqueItems: true
+                  loadBalancer:
+                    description: LoadBalancer defines how the load balancer used by
+                      the cluster is configured. LoadBalancer is available in TechPreview.
+                    properties:
+                      type:
+                        default: OpenShiftManagedDefault
+                        description: type defines the type of load balancer used by
```

Comment:
Perhaps we want to mention in the description that this is tech preview

Reply:
can we? I literally copy/pasted from the original CRD.

Reply:
The description for `loadBalancer` includes that this is TechPreview, so I think we're good here :+1: